datafusion/core/src/dataframe/mod.rs

```diff
@@ -1215,7 +1215,7 @@ impl DataFrame {
     ///   .with_param_values(vec![
     ///      // value at index 0 --> $1
     ///      ScalarValue::from(2i64)
-    ///    ].into())?
+    ///    ])?
     ///   .collect()
     ///   .await?;
     /// assert_batches_eq!(
```

Comment:
Could you please also add a demonstration of how to use the hashmap version too so that it is easier to discover by a casual user? 

Perhaps add this to the example:

```rust
    /// // Note you can also provide named parameters
    /// let results = ctx
    ///   .sql("SELECT a FROM example WHERE b = $my_param")
    ///   .await?
    ///    // replace $my_param with value 2
    ///    // Note you can also use a HashMap as well
    ///   .with_param_values(vec![
    ///      "my_param",
    ///      ScalarValue::from(2i64)
    ///    ])?
    ///   .collect()
    ///   .await?;
    /// assert_batches_eq!(

Reply:
Thanks for your comment, I'm done